src/components/LoginPage/index.js

```diff
@@ -21,20 +22,29 @@ class LoginPage extends React.Component {
       password: "",
       loading: false,
       error: "",
+      gitLoading: false,
+      feedbackMessage:"",
```

Comment:
There is no need for a new messaging element you can still use the previously existing feedback alerts

Reply:
The thing was it was there before but  it wasn' put in state 
Which was a bug so i just added it to state